src/test/Components/CreateImageWizard/steps/Users/Users.test.tsx

```diff
@@ -0,0 +1,157 @@
+import type { Router as RemixRouter } from '@remix-run/router';
+import { screen, waitFor } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+
+import { CREATE_BLUEPRINT, EDIT_BLUEPRINT } from '../../../../../constants';
+import { mockBlueprintIds } from '../../../../fixtures/blueprints';
+import { usersCreateBlueprintRequest } from '../../../../fixtures/editMode';
+import {
+  blueprintRequest,
+  clickBack,
+  clickNext,
+  enterBlueprintName,
+  getNextButton,
+  interceptBlueprintRequest,
+  interceptEditBlueprintRequest,
+  openAndDismissSaveAndBuildModal,
+  renderEditMode,
+  verifyCancelButton,
+} from '../../wizardTestUtils';
+import {
+  clickRegisterLater,
+  goToRegistrationStep,
+  renderCreateMode,
+} from '../../wizardTestUtils';
+
+let router: RemixRouter | undefined = undefined;
+
+const goToUsersStep = async () => {
+  await clickNext();
+  await clickNext();
+  await clickNext();
+  await clickNext();
+  await clickNext();
+  await clickNext();
+};
+
+const goToReviewStep = async () => {
+  await clickNext(); // Timezone
+  await clickNext(); // Locale
+  await clickNext(); // Hostname
+  await clickNext(); // Kernel
+  await clickNext(); // First boot
+  await clickNext(); // Details
+  await enterBlueprintName();
+  await clickNext(); // Review
+};
+
+const addValidUser = async () => {
+  const user = userEvent.setup();
+  const addUser = await screen.findByRole('button', { name: /add a user/i });
+  expect(addUser).toBeEnabled();
+  await waitFor(() => user.click(addUser));
+  const enterUserName = screen.getByRole('textbox', {
+    name: /blueprint user name/i,
+  });
+  const nextButton = await getNextButton();
+  await waitFor(() => user.type(enterUserName, 'best'));
+  await waitFor(() => expect(enterUserName).toHaveValue('best'));
+  await waitFor(() => expect(nextButton).toBeEnabled());
+};
+
+describe('Step Users', () => {
+  beforeEach(async () => {
+    vi.clearAllMocks();
+    router = undefined;
+  });
+
+  test('without adding user loads timezone', async () => {
+    const user = userEvent.setup();
+    await renderCreateMode();
+    await goToRegistrationStep();
+    await clickRegisterLater();
+    await goToUsersStep();
+    const addUser = await screen.findByRole('button', { name: /add a user/i });
+    expect(addUser).toBeEnabled();
+    const nextButton = await getNextButton();
+    expect(nextButton).toBeEnabled();
+    await waitFor(() => user.click(nextButton));
+    await screen.findByText('Select a timezone for your image.');
+  });
+
+  test('clicking Back loads Additional packages', async () => {
+    await renderCreateMode();
+    await goToRegistrationStep();
+    await clickRegisterLater();
+    await goToUsersStep();
+    const addUser = await screen.findByRole('button', { name: /add a user/i });
```

Comment:
Same thing here. Making sure that the back button works is a good test and something we do on the other steps. I don't think we need to check that the add user button is enabled during this specific test since that is covered in another test.